packages/language-server/src/lint-worker.ts

```diff
@@ -0,0 +1,14 @@
+import { validateSemantics } from '@neo4j-cypher/language-support';
```

Comment:
I'm aware these lint workers are now completely duplicated. It would make sense to build the worker in the language support pkg, but then we need some mechanism to copy it out to the other applications I think. I would prefer to that work at a later stage

Reply:
Can you make a cleanup card about this?

Reply:
https://trello.com/c/s3BckR3v/235-lint-worker-code-is-now-fully-duplicated